src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +58,13 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString() {
+    	if (isPrivate) {
+    		return ADDRESS_PREFIX + this.toString();
+    	} else {
+    		return ADDRESS_NOT_PRINTABLE;
+    	}
+    }
```

Comment:
👍 ... just not sure if printing `The address is not printable` will take up too much space? Especially when other fields are also not printable?  Maybe just a `***` will do?